ft_planner_node:
  ros__parameters:
    max_n_neighbors: 5
    max_dist: 6.5
    max_dist_to_first: 6.0
    max_length: 20
    threshold_directional_angle: 30
    threshold_absolute_angle: 55
    use_unknown_cones: True

    smoothing: 0.2
    predict_every: 0.1
    max_deg: 3

    maximal_distance_for_valid_path: 5
    mpc_path_length: 20
    mpc_prediction_horizon: 40

    min_track_width: 3.0
    max_search_range: 5
    max_search_angle: 45
    matches_should_be_monotonic: True
